vitrina/datasets/views.py

```diff
@@ -14,3 +14,17 @@ class DatasetListView(ListView):
 class DatasetDetailView(DetailView):
     model = Dataset
     template_name = 'vitrina/datasets/detail.html'
+    context_object_name = 'dataset'
+
+    def get_context_data(self, **kwargs):
+        context_data = super().get_context_data(**kwargs)
+        dataset = context_data.get('dataset')
+        extra_context_data = {
+            'tags': dataset.tags.replace(" ", "").split(',') if dataset.tags else "",
```

Comment:
Could you move this logic out of view into `Dataset` model? We want to keep all data mangling close to data layer. Otherwise refactoring will be difficult, if we don't keep things in one place.